src/common/util/obfuscate.js

```diff
@@ -8,10 +8,6 @@ var fileProtocolRule = {
   replacement: 'file://OBFUSCATED'
 }
 export class Obfuscator extends SharedContext {
-  constructor (parent) {
-    super(parent) // gets any allowed properties from the parent and stores them in `sharedContext`
-  }
-
   shouldObfuscate () {
     return getRules(this.sharedContext.agentIdentifier).length > 0
```

Comment:
`this.sharedContext.agentIdentifier` will be undefined without the constructor passing args into super

Reply:
Is this not superfluous since there's a [default constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor#description) if one isn't provided

Reply:
looks like it, TIL